payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/PaymentServiceImpl.java

```diff
@@ -33,6 +42,20 @@ public WalletEntity makePayment(final WalletPayment walletPayment) {
         final var updatedToWalletBalance = toWallet.getBalance().add(walletPayment.amount());
         updateBalance(updatedToWalletBalance, toWallet);
 
+        final HttpHeaders httpHeaders = new HttpHeaders();
+        httpHeaders.setContentType(MediaType.APPLICATION_JSON);
+
+        final Map<String, String> requestBody = new HashMap<>();
+        requestBody.put("name_from", fromWallet.getName());
+        requestBody.put("id_from", String.valueOf(walletPayment.walletIdFrom()));
+        requestBody.put("name_to", toWallet.getName());
+        requestBody.put("id_to", String.valueOf(walletPayment.walletIdTo()));
+        requestBody.put("amount", walletPayment.amount().toString());
+
+        final HttpEntity<Map<String, String>> requestEntity = new HttpEntity<>(requestBody, httpHeaders);
+
+        restTemplate.postForEntity(PathUtils.TRANSACTION_HISTORY_PATH, requestEntity, String.class);
+
```

Comment:
You have to extract logic of sending data to  `client/TransactionHistoryClient.java`